### DISCUSSION TITLE
Dashboard service groups and service names

### DISCUSSION THREAD

**Discussion by mkannance** [apache/skywalking#12124] [Q&A] [answered]
Hi Team, I have configured monitoring successfully, my request is service groups name showing in the service name. but demo skywalking page showing neat and clear, I want the same name framing in the agent configuration, I have attached demo page and my own page for your reference. Kindly help on this. thanks in advance.
![demoapps](https://github.com/apache/skywalking/assets/59562078/df8d978a-5dfa-4cd6-991b-ee5770daaf9d)
![ownapp](https://github.com/apache/skywalking/assets/59562078/a26a2718-935f-41db-97b6-c17df3eeeb69)

**Comment by mkannance** [comment]
Existing setup

Service groups: Homes
Service names: Homes::secure

My expectation is 

Service groups: Homes
Service names: Secure

**Comment by wu-sheng** [comment]
The latest one has improved this. UI and backend did that. 

No agent side change is required.

**Reply by mkannance** [reply]
Thanks for your response.  May I configure  Service Groups Like DEV, UAT, PROD instead of name as agent.

**Reply by wu-sheng** [reply, marked as answer]
It is just a name prefix in the service name. Nothing special.

https://skywalking.apache.org/docs/main/latest/en/setup/backend/service-auto-grouping/


No matter what you will use, note, we don't care and don't process that in another way.

**Reply by mkannance** [reply]
Thank you so much for your guidance.
